app/controllers/API/V1/posts_controller.rb

```diff
@@ -1,7 +1,11 @@
 class Api::V1::PostsController < ApplicationController
   load_and_authorize_resource
+  skip_before_action :authenticate_user!
+  before_action :authenticate_request
+  after_action :hard_logout
+
   def index
-    @posts = Post.all.order('created_at')
+    @posts = User.find(params[:user_id]).posts.order('created_at')
```

Comment:
* Even after adding a bearer token I still get the error reported by the previous CR kindly look into it 
![image](https://github.com/Graciano1997/technology_blog_app/assets/52497246/4c1197f7-0856-41e2-96fd-9744b9d8de76)